specs/agents/logging.md

```diff
@@ -0,0 +1,36 @@
+## Agent logging
+
+### `log_level` configuration
+
+Sets the logging level for the agent.
+
+This option is case-insensitive.
+
+|                |   |
+|----------------|---|
+| Valid options  | `trace`, `debug`, `info`, `warning`, `error`, `critical`, `off` |
+| Default        | `info` |
+| Dynamic        | `true` |
+| Central config | `true` |
```

Comment:
Not supported at the moment in the RUM agent, But we do support changing the logLevel dynamically. 

One of the reasons for not adding the config options in the RUM agent is as we don't have a good AUTH story.

Reply:
Good to know. Hopefully some day!